### ISSUE TITLE
Github pages not working properly

### ISSUE THREAD

**Issue by jonasskanoy** [alshedivat/al-folio#458]
**Describe the bug**
It seems like the gh page is giving me som issue with the sass functions of the website. I forked the al-folio repo, and followed the instructions as shown.

**Expected behavior**
Expected same result on the gh page as on the local run.

**Screenshots**
Website with github pages:
![image](https://user-images.githubusercontent.com/71043573/142027790-00fccbd4-b823-4781-86ed-37064a3674d2.png)

Website with local run instance:
![image](https://user-images.githubusercontent.com/71043573/142027880-4a20e345-8a7b-4ddc-aac1-d4273eb67737.png)

**System:**
- OS: Windows 10, running wsl
- Browser: Brave v.1.31.91
- Jekyll version: 4.2.1
- Ruby version: 2.7.0p0

**Comment by jmrplens** [comment]
Try this.
In config.yml leave empty the base url

**Comment by varunjain3** [comment]
I m also facing a similar issue where I am trying to deploy the page. I left the base_url empty as show below: 

https://github.com/varunjain3/varunjain3.github.io/blob/439548f88397069b35dfb14c15754eb94de1d216/_config.yml#L18-L19

The deployment says that Github Pages has issues 
```
GitHub Pages failed to build your site.

The tag responsive_image on line 5 in _posts/2015-05-15-images.md is not a recognized Liquid tag.
````
Where m I going wrong?

**Comment by jonasskanoy** [comment]
> Try this. In config.yml leave empty the base url
When I tried that the page looked like this:
![image](https://user-images.githubusercontent.com/71043573/142167619-c2b2ae80-ac05-421c-be9c-ea4d8cb33e02.png)

**Comment by alshedivat** [comment]
please follow instructions on how to deploy the website: https://github.com/alshedivat/al-folio#deployment

**Comment by Z-Y00** [comment]
I got the same problem of "The tag `responsive_image_block` on line 14 in `/_layouts/about.html` is not a recognized Liquid tag."
I guess this might be related sth wrong with the recent commit
https://github.com/alshedivat/al-folio/pull/439

**Comment by Z-Y00** [comment]
I tried to build the page without any modification. But it failed.

The tag `responsive_image` on line 5 in `_posts/2015-05-15-images.md` is not a recognized Liquid tag. For more information, see https://docs.github.com/github/working-with-github-pages/troubleshooting-jekyll-build-errors-for-github-pages-sites#unknown-tag-error.

My test branch is here
https://github.com/Z-Y00/academic/commits/test

**Comment by varunjain3** [comment]
Even after reading deployment page, i m still facing the problem

**Comment by Z-Y00** [comment]
For the broken picture, in my case, I believe the whole problem is due to the **miss of baseurl**
You can see this at my page 
The generated webpage point to:
https://z-y00.github.io/assets/img/prof_pic.jpg
But the actual picture is here:
https://z-y00.github.io/academic/assets/img/prof_pic.jpg
academic is the baseurl
@alshedivat I guess you can confirm the issue at my webpage.
https://z-y00.github.io/academic/

For the tag responsive_image, I also met the same problem @varunjain3 
 I found that if it is deployed via Action and gh-pages branch, then it works out fine.

**Comment by alshedivat** [comment]
@Z-Y00, thanks for pointing out this. I believe the problem might be related to #451 (all images on the demo page were broken for a short while). should be fixed in #468. please make sure your `_includes/responsive_img.html` is identical to https://github.com/alshedivat/al-folio/blob/b45b2475018cda946a1ea825ef408118db504540/_includes/responsive_img.html#L1-L8

**Comment by Z-Y00** [comment]
Thanks for your help! It works! 
After the revert of "remove extra space", everything looks good now.
